apps/librelingo_json_export/librelingo_json_export/challenges.py

```diff
@@ -57,7 +58,12 @@ def _get_phrase_challenges(phrase, course: Course):
 def _get_word_challenges(word, course: Course):
     "Generate challenges based on a Word"
     return _challenge_mapper(
-        [get_cards_challenge, get_short_input_challenge, get_listening_challenge]
+        [
+            get_cards_challenge,
+            get_short_input_challenge,
+            get_listening_challenge,
+            get_pairing_challenge,
```

Comment:
I realized that this is wrong for a profound reason. 

This is my fault because I didn't remember correctly how these things actually work. Basically, all current challenges work based on individual words or phrases, but this new one doesn't.

I'm working on a change in order to make this easier to work with.

Reply:
I was thinking that was an issue, which I think I tried to create pairs and the odd list thing to work around the fact that only a single word/phrase could be passed through. Thank you for working on a change to make it a little easier to work on!

Reply:
I made some changes and now it seems to work well. There is still some tests that fail but that seems like is because they simply need to be updated to include this new challenge type.

I pushed my changes into this branch on my repo: https://github.com/LibreLingo/LibreLingo/commits/word-pairing-challenge

i. e. the same branch name as you have in your fork. You can merge my changes into your fork in order to get my changes on your local branch. (it should be sth like `git merge upstream/word-pairing-challenge`)

in order to do this probably you need to have `upstream` (or any other name that you choose) which is the git link for my version of the repo. If you don't have the `upstream` remote server set up yet,  the way you can do that is this command `git remote add upstream git@github.com:LibreLingo/LibreLingo.git`